requirements/requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy~=1.16
+numpy~=1.16,<1.24  # upper bound added since numpy==1.24 broke importing mxnet, see https://github.com/awslabs/gluonts/pull/2506
```

Comment:
```suggestion
# upper bound added since numpy==1.24 broke importing mxnet,
# see https://github.com/awslabs/gluonts/pull/2506
numpy~=1.16,<1.24  
```